docs/source/worked-examples-openstack.rst

```diff
@@ -0,0 +1,112 @@
+Worked Examples: Using Calico-based OpenStack
+=============================================
+
+Once you've installed OpenStack (see :doc:`openstack`) and configured its
+networking (see :doc:`opens-external-conn`) you'll want to let your users start
+playing with your brand new cloud.
+
+Before you do, it will be helpful to have a few worked examples for common
+deployment scenarios. In particular, this will make it easy for you to set up
+topologies and examine their connectivity to try to get an understanding of the
+way Calico networks behave.
+
+This document contains several sections, each of which demonstrates a
+representative deployment model for a kind of workload. The early examples are
+simple, containing small numbers of machines, scaling to the later examples
+that demonstrate quite complex arrangements. Each example contains diagrams
+that represent a connectivity graph, to add clarity to the textual
```

Comment:
Arguably, if you remove this statement about the diagrams, then we could publish this now and just add to it over time.  In other words, I think we have information here, which would already be better published than hidden on github...